docs/dev/benchmark.rst

```diff
@@ -0,0 +1,112 @@
+| software\_config      | {"os": "ubuntu:18.04","pip": { "docker": "4.1.0", "gitpython": "3.0.4", "numpy": "1.17.4", "onnx": "1.6.0"}}                                               | backend      | env dependency list                                                          |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------+------------------------------------------------------------------------------+
+| hardware\_config      | "mbp13inch2019"                                                                                                                                            | hardware     | descriptor of target hardware environment, optionally json                   |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------+------------------------------------------------------------------------------+
+| runtime\_config       | {“num\_cpu\_threads”: 3, “cudnn”: “cudnn-8”, “cuda\_driver”: “480.10.1”, “os”: linux}                                                                      | hardware     | backend runtime arguments, optionally specified                              |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------+------------------------------------------------------------------------------+
+| execution\_config     | {“number”: 1, “repeat”: 10, “min\_repeat\_ms”, 0}                                                                                                          | statistics   | workload execution parameters                                                |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------+------------------------------------------------------------------------------+
+| statistics            | {“runtime\_ms\_mean”: 12,“runtime\_ms\_std”: 12}                                                                                                           | statistics   | all metrics, other optional fields ``binary_size``, ``compile_time``, etc.   |
++-----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------+------------------------------------------------------------------------------+
+
+Sample data: 
```

Comment:
We need to talk about possible formats. Right now it is json line which is fine, but we should also mention othe r possible compatible format.